src/chocolatey.install/chocolatey.wxs

```diff
@@ -23,10 +23,6 @@
     <Property Id="ARPNOREPAIR" Value="yes" Secure="yes" />      <!-- Remove repair from post-install options, as it will not work with the script method -->
     <Property Id="ARPNOMODIFY" Value="yes" Secure="yes" />      <!-- Remove modify from post-install options, as it will not work with the script method -->
 
-    <CustomAction Id="SetHelpersDirectoryPath" Property="HelpersDirectoryPath" Value="[INSTALLDIR]\\helpers" />
-    <CustomAction Id="SetRedirectsDirectoryPath" Property="RedirectsDirectoryPath" Value="[INSTALLDIR]\\redirects" />
-    <CustomAction Id="SetToolsDirectoryPath" Property="ToolsDirectoryPath" Value="[INSTALLDIR]\\tools" />
-
```

Comment:
Why are you removing these custom actions?

I was under the impression they are required to be available?

Reply:
These custom actions only fire when Chocolatey is already installed, which only happens on an upgrade, change, repair, or uninstall. None of which we currently do anything with.